datasets/tglc.yaml

```diff
@@ -0,0 +1,33 @@
+Name: TESS-GAIA Light Curve (TESS)
+Description: |
+  TESS-Gaia Light Curve (TGLC) is a PSF-based TESS full-frame image (FFI) light curve product. Using Gaia DR3 as priors, the team forward models the FFIs with the effective point spread function to remove contamination from nearby stars. The resulting light curves show a photometric precision closely tracking the pre-launch prediction of the noise level: TGLC's photometric precision consistently reaches ≲2% at 16th TESS magnitude even in crowded fields, demonstrating excellent decontamination and deblending power.
+Documentation: https://archive.stsci.edu/hlsp/tglc
+Contact: archive@stsci.edu
+ManagedBy: "[Space Telescope Science Institute](http://www.stsci.edu/)"
+Citation: Please cite [Han & Brandt 2023](https://ui.adsabs.harvard.edu/abs/2023AJ....165...71H/abstract) when using TGLC data in published works.
+UpdateFrequency: Monthly
+Tags:
+  - astronomy
+  - aws-pds
+License: All HLSPs hosted at MAST are subject to a [CC By 4.0 license](https://creativecommons.org/licenses/by/4.0/).
+Resources:
+  - Description: TESS Mission data files
```

Comment:
I know next to nothing about AWS registries, but since this is an HLSP and not an official mission product, should the description be different?

Reply:
ope. it definitely should be. I copied this file from TESS, this is a holdover